debs/cloud-info-provider-opennebula/debian/copyright

```diff
@@ -0,0 +1,34 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: cloud-info-provider
+Source: https://github.com/EGI-FCTF/BDIIscripts
+
+Files: *
+Copyright: 2014 Salvatore Pinto, Álvaro Lopez,
+                Boris Parak, Björn Hagemeier,
+                Enol Fernández
```

Comment:
@gwarf you are missing and probably this should be EGI Foundation, CSIC, FZJ instead, also years are wrong.

This is related to #76, so we could fix everything in a coming PR instead of partially patching things now.

Reply:
Thanks @enolfc, this `copyright` file is a copy of the existing one, and indeed it needs some update and should be addressed together with #76